### ISSUE TITLE
pue.php: how to calculate category 'Other'?

### ISSUE THREAD

**Issue by spezialist1** [MorinPierre/openDCIM#31]
Pierre, as far as I understand, now the category 'Other' (on graphics of PUE with color wheat) is not calculated in any way?

Very thanks for the support.

**Comment by MorinPierre** [comment]
The line Other is the sum of UPS Outputs + IT, Cooling and Other Mechanical which are not powered by an UPS.
Those graphs can be a bit confusing as they don't represent values of each ElectricalMP categories but intervals between them. Here is a short description of what is calculated:

-Power Sources loss = UPS Input + (IT, Cooling and Other Mechanical which are not powered by an UPS)
-Other = UPS Outputs + (IT, Cooling and Other Mechanical which are not powered by an UPS)
-Other Mechanical Devices = IT + Cooling + Other Mechanical
-Cooling = IT + Cooling;
-IT = IT
PUE = Power Sources loss / IT

**Comment by spezialist1** [comment]
Aha, then for me everything clears up, thanks for the detailed answer.

Tell, please, and whether PUE will be correctly calculated, if at me the quantity of UPS Input is not equal to quantity of UPS Output?

**Comment by MorinPierre** [comment]
As PUE calculation doesn't use UPS Output values, it won't affect it. The warning saying that the quantity of Input and Output are different is just a warning. If you consider that it's normal in your configuration to have different quantities, you can do it. The code just signals it in case you missed something.

**Comment by spezialist1** [comment]
Aha, I understood, 'Power Sources loss' is a power consumption for all remaining auxiliary needs of DC, i.e., actually are losses of energy, and it is bad.

After some setup, here that it is possible to see within a week:
![my_pue](https://cloud.githubusercontent.com/assets/10425179/9380006/f9cb1cec-4735-11e5-832b-cd58cc1d3590.png)

I think that for a month and more I will see more realistic result :-).

**Comment by spezialist1** [comment]
Pierre, explain please how to use 'Penalty' on DCEM page?

Very thanks for the support.

**Comment by MorinPierre** [comment]
Penalties allow to represent other losses (like heat) as described in http://www.etsi.org/deliver/etsi_es/205200_205299/2052000201/01.02.01_60/es_2052000201v010201p.pdf. I recommend to read this document to better understand DCEM calculation.

**Comment by spezialist1** [comment]
Ok, thanks for the pdf-link.
